### PR TITLE
Add disabled item functionality

### PR DESCRIPTION
**Pull request by mikeymike**

### CODE REVIEW

File: src/MenuItem/SelectableTrait.php [hunk 1]
```diff
@@ -24,6 +24,11 @@
     private $showItemExtra = false;
 
     /**
+     * @var bool
+     */
+    private $disabled;
```
Comment (AydinHassan) [inline src/MenuItem/SelectableTrait.php]: should have a default value

File: src/MenuStyle.php [hunk 2]
```diff
@@ -138,6 +139,7 @@ class MenuStyle
      * @param TerminalInterface $terminal
      * @throws InvalidInstantiationException
      */
+    // TODO: Disabled marker ideally without breaking change.
```
Comment (AydinHassan) [inline src/MenuStyle.php]: remove

**Comment by mikeymike** [comment]
@AydinHassan Updated dude... Just grabbing some food then I'll release a new version 👍

**Comment by AydinHassan** [comment]
LGTM @mikeymike, merge and tag away when you're ready

**Comment by AydinHassan** [comment]
🎉

**Pull request merged**